catalog/MDCCatalog/MDCCatalogComponentsController.swift

```diff
@@ -243,7 +243,11 @@ class MDCCatalogComponentsController: UICollectionViewController, MDCInkTouchCon
                                    left: Constants.inset + view.safeAreaInsets.left,
                                    bottom: Constants.inset,
                                    right: Constants.inset + view.safeAreaInsets.right)
-    titleLabel.superview!.removeConstraints(titleLabel.superview!.constraints)
+
+    // Remove the leading, trailing, bottom constraints
+    titleLabel.superview!.constraints.forEach({ $0.isActive = false })
+    // Remove the height constraint
+    titleLabel.constraints.forEach({ $0.isActive = false })
```

Comment:
This seems ok as a quick fix, but would it be safer to keep a reference to the constraint so that we can enable/disable/change it at will?

Reply:
It would definitely be safer and it's my preferred approach. However, the existing code uses this approach and since I didn't know why it did, I felt it best to keep the change as close to the original.  I can track the height constraint and simply adjust that instead.

Reply:
sgtm - feel free to do the clean up in a follow-up PR or file an issue if you want to punt.